.github/workflows/build.yml

```diff
@@ -34,6 +42,9 @@ jobs:
       - name: Build
         run: npm run build
 
+      - name: Index algolia
```

Comment:
The `build` workflow is only to check if the documentation compiles, it should not update algolia index.

We should either remove this step or run in in some kind of dry-run mode (e.g. only to test if the parser for headers work etc. but not send the update to algolia yet)

Reply:
Removed, please check again

Reply:
I still can see this in the `build` workflow, but don't see it now on the `nodejs` one. The `nodejs` is publishing the github page and this is where we should update algolia index.

Reply:
yes, I believe that building algolia index should be in nodejs workflow for sure.